tensorflow/docs_src/install/install_linux.md

```diff
@@ -171,7 +171,7 @@ Take the following steps to install TensorFlow with Virtualenv:
      issue the following command to install TensorFlow in the active
      virtualenv environment:
 
-     <pre>(tensorflow)$ <b>pip3 install --upgrade \
```

Comment:
In this file, the above version needs to be updated.
The pip package version below is 3.4 so chancing this wont fix the documentation.